src/client/calendar/utils/get-calendar-slug.js

```diff
@@ -0,0 +1,3 @@
+const getCalendarSlug = calendar => calendar.id.replace('cal-', '')
```

Comment:
@graymur this function doesn't require entire calendar then, since it only uses `id`

Reply:
What if this changes in the future? I think it's quite possible.

Reply:
Why do you think so? But if it does we will refactor.